.github/workflows/e2e.yml

```diff
@@ -5,9 +5,6 @@ on:
       - changeset-release/*
   workflow_dispatch:
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Did you consider using [`jobs.<job_id>.strategy.fail-fast: false`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) instead of disabling the concurrency limiter?

Reply:
Thanks for hint - I've checked  and it works as it should 🙇🏻